doc/user-guide/dask.rst

```diff
@@ -2,195 +2,151 @@
-    writing netCDF files with Dask's distributed scheduler is only supported for
-    the `netcdf4` backend.
+        xds = rioxarray.open_rasterio("my-satellite-image.tif", chunks='auto')
 
-A dataset can also be converted to a Dask DataFrame using :py:meth:`~xarray.Dataset.to_dask_dataframe`.
+    See :ref:`io.rasterio` for more details.
 
-.. ipython:: python
-    :okwarning:
 
-    df = ds.to_dask_dataframe()
-    df
+Loading Dask Arrays
+~~~~~~~~~~~~~~~~~~~
 
-Dask DataFrames do not support multi-indexes so the coordinate variables from the dataset are included as columns in the Dask DataFrame.
+.. ipython:: python
+    :suppress:
 
+    import os
 
-Using Dask with xarray
-----------------------
+    import numpy as np
+    import pandas as pd
+    import xarray as xr
+
+    np.random.seed(123456)
+    np.set_printoptions(precision=3, linewidth=100, threshold=100, edgeitems=3)
 
-Nearly all existing xarray methods (including those for indexing, computation,
-concatenating and grouped operations) have been extended to work automatically
-with Dask arrays. When you load data as a Dask array in an xarray data
-structure, almost all xarray operations will keep it as a Dask array; when this
-is not possible, they will raise an exception rather than unexpectedly loading
-data into memory. Converting a Dask array into memory generally requires an
-explicit conversion step. One notable exception is indexing operations: to
-enable label based indexing, xarray will automatically load coordinate labels
-into memory.
+    ds = xr.Dataset(
+        {
+            "temperature": (
+                ("time", "latitude", "longitude"),
+                np.random.randn(30, 180, 180),
+            ),
+            "time": pd.date_range("2015-01-01", periods=30),
+            "longitude": np.arange(180),
+            "latitude": np.arange(89.5, -90.5, -1),
+        }
+    )
+    ds.to_netcdf("example-data.nc")
 
-.. tip::
+There are a couple common cases where you may want to convert lazy Dask arrays into eager, in-memory Xarray data structures:
 
-   By default, dask uses its multi-threaded scheduler, which distributes work across
-   multiple cores and allows for processing some datasets that do not fit into memory.
-   For running across a cluster, `setup the distributed scheduler <https://docs.dask.org/en/latest/setup.html>`_.
+- You want to inspect smaller intermediate results when working interactively or debugging
+- You've reduced the dataset (by filtering or with a groupby, for example) and now have something much smaller that fits in memory
 
-The easiest way to convert an xarray data structure from lazy Dask arrays into
-*eager*, in-memory NumPy arrays is to use the :py:meth:`~xarray.Dataset.load` method:
+To do this, you can use :py:meth:`~xarray.Dataset.load`, which is similar to :py:meth:`~xarray.Dataset.compute`, but instead changes results in-place:
 
 .. ipython:: python
 
     ds.load()
```

Comment:
Let's emphasize `.compute()` (and `.persist()` instead of `.load()` (which is a strange method that modifies an xarray object in place)

Reply:
Is there a reason to mention `load()`? I added a note that it's recommended people use `compute()` instead